File: references/MWORKS高校星火计划资料包/培训课程配套材料/01-官网课程配套材料/00-快速入门课程/02-Syslab快速入门/05-Julia语法-入门/附件示例/对应课程PPT页码/07_与其他语言的差异/07-1Julia与M语言的显著差异/page_252/page_252.jl
function addme2(a, b)
  result = a + b
  absResult = abs(result)
  return result, absResult
end
addme2(1, -3)
# 返回元组 (-2, 2)
x, = addme2(1, -3)
# x = -2
x, y = addme2(1, -3)
# x = -2, y = 2

File: references/MWORKS高校星火计划资料包/培训课程配套材料/01-官网课程配套材料/01-数学与基础应用/03-Julia语法-入门/附件示例/对应课程PPT页码/07_与其他语言的差异/07-1Julia与M语言的显著差异/page_252/page_252.m
% 文件1
function [result,absResult] = addme2(a,b)
    result = a+b
    if nargout > 1
        absResult = abs(result);
    end
end
% 文件2 或 在命令行窗口输入
addme2(1,-3)
% 返回 -2
[x,y]=addme2(1,-3)
% x = -2, y = 2



